src/Aspire.Hosting.Python/api/Aspire.Hosting.Python.cs

```diff
@@ -30,7 +30,7 @@ namespace Aspire.Hosting.Python
 {
     public partial class PythonAppResource : ApplicationModel.ExecutableResource, IResourceWithServiceDiscovery, ApplicationModel.IResourceWithEndpoints, ApplicationModel.IResource
     {
-        public PythonAppResource(string name, string executablePath, string projectDirectory) : base(default!, default!, default!) { }
+        public PythonAppResource(string name, string executablePath, string appDirectory) : base(default!, default!, default!) { }
```

Comment:
Source breaking change if somebody uses named arguments. Not a big deal.

Reply:
Yeah it's preview so OK to do, will add a doc issue.

Reply:
https://github.com/dotnet/docs-aspire/issues/2559